www/src/components/pagination/index.js

```diff
@@ -92,6 +93,8 @@ class Pagination extends React.Component {
               <option
                 value={`${i === 0 ? `` : i + 1}`}
                 key={`pagination-number${i + 1}`}
+                aria-label={`Goto Page ${i + 1}`}
+                aria-current={currentPage === i + 1 && `page`}
```

Comment:
Why do you need the `` && `page` ``?
```suggestion
                aria-current={currentPage === i + 1}
```

Reply:
Wondered that myself, since I didn't notice a difference when using the NVDA screenreader. Took the recommendation from https://tink.uk/using-the-aria-current-attribute/
Search for 'page' there. It's under the header: The aria-current attribute.

PS: Yay, nen anderen Belg, hoera, hoera!

Reply:
We don't need the `&& `page``

@NickyMeuleman Can you apply the suggestion? I'll go ahead and merge 🙂

Reply:
done :)